plugins/zammad-mcp/references/mcp-builder/scripts/connections.py

```
@@ -0,0 +1,149 @@
+        super().__init__()
+        self.url = url
+        self.headers = headers or {}
+
+    def _create_context(self):
+        return sse_client(url=self.url, headers=self.headers)
+
+
+class MCPConnectionHTTP(MCPConnection):
+    """MCP connection using Streamable HTTP."""
+
+    def __init__(self, url: str, headers: dict[str, str] = None):
```

Ruff (0.15.12)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)